From e11d039c50d13ac638c2ef77285af3543805a2f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mantas=20Mikul=C4=97nas?= <grawity@gmail.com>
Date: Wed, 30 Apr 2014 19:53:13 +0300
Subject: [PATCH] man: update journald rate limit defaults

This brings the man page back into sync with the actual code.

(cherry picked from commit 8f18f550e7023948f199616fdfbb0f09711fd615)

Resolves: #1145352
---
 man/journald.conf.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/journald.conf.xml b/man/journald.conf.xml
index b362c5d..87e13b2 100644
--- a/man/journald.conf.xml
+++ b/man/journald.conf.xml
@@ -190,8 +190,8 @@
                                 limiting is applied per-service, so
                                 that two services which log do not
                                 interfere with each other's
-                                limits. Defaults to 200 messages in
-                                10s. The time specification for
+                                limits. Defaults to 1000 messages in
+                                30s. The time specification for
                                 <varname>RateLimitInterval=</varname>
                                 may be specified in the following
                                 units: <literal>s</literal>,
